modules/components/src/AdvancedFacetView/LiveAdvancedFacetView.js

```diff
@@ -124,8 +124,22 @@ export default class LiveAdvancedFacetView extends React.Component {
     }
   }
   onSqonFieldChange = ({ sqon }) => {
-    const { onSqonChange = () => {} } = this.props;
-    this.setState({ sqon }, () => onSqonChange({ sqon }));
+    const {
+      onSqonChange = () => {},
+      projectId: PROJECT_ID,
```

Comment:
can  you update the code to use `projectId` instead of renaming it here? (same for PROJECT_ID and ES_INDEX in the rest of the file)